lib/ssh_scan/api.rb

```diff
@@ -104,6 +104,13 @@ class API < Sinatra::Base
           "#{params[:target]}:#{params[:port] ? params[:port] : "22"}"
         options[:policy_file] = options[:policy]
         options[:uuid] = SecureRandom.uuid
+        settings.count_req += 1
+        if (Time.now - settings.curr_time) > 60.0
+          settings.requests_per_min = 0
+          settings.curr_time = Time.now
+        else
+          settings.requests_per_min += 1
+        end
```

Comment:
I think we should have a Ruby object class that tracks requests in a unified way.  I feel like that might make the integration a bit cleaner that splicing the raw code into the API route code as is.  It could simply act as a non-blocking gateway for new scan requests (Example `Stats.new_scan_request`) and it could internally manage all the attributes we care about and also be easily unit-tested outside of the API context. (Example `Stats.requests_per_min`, `Stats.avg_requests_per_min`, `Stats.total_requests`)

As for the stats that require information from the DB, we could add a method off of those to generate the stats for us or we could use the Stats interface to call the DB (Example `db.avg_worker_time`) and retrieve those details.  DB might be the most straight forward.